runtime/asset_font_selector.cc

```diff
@@ -109,8 +119,10 @@ AssetFontSelector::FlutterFontAttributes::~FlutterFontAttributes() {}
 void AssetFontSelector::parseFontManifest() {
   std::vector<uint8_t> font_manifest_data;
   if (!directory_asset_bundle_->GetAsBuffer(kFontManifestAssetPath,
```

Comment:
Since the `AssetFontSelector` is now initialized with either a `DirectoryAssetBundle` or a `ZipAssetStore`, don't we need null checks on `directory_asset_bundle_` and `asset_store_`?